src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -311,7 +311,6 @@ public function createAction(Request $request)
      * Show Employee edit page.
      *
      * @DemoRestricted(redirectRoute="admin_employees_index")
-     * @AdminSecurity("is_granted('update', request.get('_legacy_controller'))", redirectRoute="admin_employees_index")
```

Comment:
I removed this security rule that was added in 1.7.7, since all users should be able to access their profile edition page in BO.  What do you guys think?